### DISCUSSION TITLE
Line-up planning for the current week

### DISCUSSION THREAD

**Discussion by vulcan87** [ho-dev/HattrickOrganizer#2180] [Q&A]
Hi all,

it's my first week using the HO for layout planning. So I'm a bit confused because it's hard to plan.

1) There is a marker in front of the names if the player has played in a training position. It seems that the data of the past week is displayed there. How can I get a quick overview of which player is already set for the current week? (After creating the lineup for the league game, I can't find where I can see which player is already in a training relevant lineup this week)

![ho_lineup_ideal First](https://github.com/user-attachments/assets/d0040afd-8f59-46c4-b36b-dff455c78764)

2) The setup assistant. If I tick the ‘Ideal Position First’ box, it still does not use the assigned best positions (Screenshot One).
It gets even worse if I tick ‘Exclude last line-up’. Then the players are mixed up even more and players are still used who are already set this week (Screenshots: 1 - result of the wizard for the friendly match, 2 - saved line-up for the league)

If the checkmark is set, it will purge all players even if I don't use the wizard (so players will disappear in the league game, I have to pull it from Hattrick if i will see the saved lineup). If I uncheck the box, download the league settings, switch to the friendly and check the ‘exclude last’ box, all players from the leage game are removed, including them which are set as substitute.

![ho_aufstellung_ideal letzte aufstellung ignorieren](https://github.com/user-attachments/assets/da9acff4-a27b-4471-b095-ea823bae553d)
![HO_lineup_league](https://github.com/user-attachments/assets/eaf481fa-062e-4d7c-9c5c-cda2ddd08f1f)

How did I have to use the HO for planning the current week? 

Best wishes
Reno

**Comment by immertrainer** [comment]
Hallo!
ich bin glaube ich etwas zu doof, aber wie kann ich den HattrickOrganizer überhaupt herunterladen und zum laufen bringen?

**Reply by vulcan87** [reply]
Der link wurde dir kurz ja schon bei dem Issue geschickt.  Nach der Installation sollte die CHPP abfrage kommen und dann läuft er eigentlich.

**Reply by vulcan87** [reply]
Der link wurde dir kurz ja schon bei dem Issue geschickt.  Nach der Installation sollte die CHPP abfrage kommen und dann läuft er eigentlich.
